amr-wind/equation_systems/BCOps.H

```diff
@@ -39,7 +39,11 @@ struct BCOp<PDE, std::enable_if_t<std::is_base_of_v<ScalarTransport, PDE>>>
     {
         amrex::IntVect ng_diff(1);
         auto& field = m_fields.field;
-        field.fillphysbc(m_time.new_time(), ng_diff);
+        // Currently, this is only called from old or new states
```

Comment:
instead of a comment, maybe an abort if it is anything other than old or new? Maybe that's too tricky...